app/representers/cms/page_representer.rb

```diff
@@ -1,32 +1,28 @@
+# frozen_string_literal: true
+
 module CMS::PageRepresenter
   include ThreeScale::JSONRepresenter
 
-  wraps_resource
+  wraps_resource ->(*) { self.class.data_tag }
 
   property :id
-
-  with_options(if: ->(options) { options[:short] == false }) do |p|
-    p.property :draft, render_nil: true
-    p.property :published, render_nil: true
-  end
-
+  property :created_at
+  property :updated_at
+  property :title
   property :system_name
-  property :liquid_enabled
+  property :layout_id
 
   with_options(if: ->(*) { is_a?(CMS::Page) }) do |p|
-    # this needs a getter
+    p.property :section_id
     p.property :path, if: ->(*) { respond_to?(:path) }
-    p.property :title
-    p.property :hidden
-    p.property :layout
     p.property :content_type
+    p.property :liquid_enabled
```

Comment:
The property is not returned when `false`. I think this could work (haven't verified myself)

```suggestion
    p.property :liquid_enabled?
```